### ISSUE TITLE
StarRocks: Use FILES to bulk load

### ISSUE THREAD

**Issue by flarco** [slingdata-io/sling-cli#146]
https://docs.starrocks.io/docs/sql-reference/sql-functions/table-functions/files/

**Comment by alberttwong** [comment]
Yes.. the issue is that files is only for parquet.   Doesn't still work with JSON and CSV yet.    It'll take some time so thats why I didn't want you to waste your engineering time and just do inserts for now and then when it gets all moved to files(), then we can refactor for performance.

**Comment by alberttwong** [comment]
Also behind the scenes, when you do single inserts in the StarRocks front end, we batch them up and stream load it.   So basically our MySQL layer is forced insert micro batches.
